lib/githubapi.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function githubapi(github) {
```

Comment:
What's the type of the `github` argument? Add a jsDoc comment to make it clear to the reader